reference/product_information/index.rst

```diff
@@ -0,0 +1,93 @@
+
+The following chapter is designed for developers (integrators and contributors) and provides all information needed to create quick overview and understanding of how product data is kept in **PIM**.
+
+Overview
+--------
+
+The data model is structured as Entity - Attribute - Value (i.e. Product - Attribute - ProductValue).
+
+The **Product** entity (``Pim\Component\Catalog\Model\Product``) has many **ProductValues** (``Pim\Component\Catalog\Model\ProductValue``) that are linked to different **Attributes** and hold value information depending on specific attribute they are related too.
+
+Product
+-------
+
+The main entity is the product which,
+
+* has values;
+* belongs to a family;
+* is positioned in many categories.
+
+.. note::
+    For information on how to import/export categories using CSV format see :doc:`/reference/import_export/formats/category`.
+
+
+.. note::
+    PIM architecture is very flexible and allows you clear and easy way to customize the catalog structure for your own integration: :doc:`/cookbook/catalog_structure/index`
+Product value
+-------------
+
+Each product value is linked to a product, an attribute and some additional entities related to (media, metrics, options, etc).
+Each product **has many** product values that represent information about it according to their attribute type. 
+
+.. note::
+    For information on how to import/export product using CSV format see :doc:`/reference/import_export/formats/product`. 
+
+Attribute
+---------
+
+The attribute determines the data type and properties of a product value according to its type definition.
+
+Attributes are combined in attribute groups.
+
+An attribute with an activated localized property has different values for each locale, eg: French and English
+
+A scoped attribute has different value for each channel, eg: e-commerce and mobile
+
+An attribute can be localized and scoped, and therefore have different values for each combination of locale and
+channel, eg: e-commerce - French, e-commerce English, etc.
+
+Attribute type
+--------------
+
+Any attribute belongs to attribute type that determines its definition and product value data type and definition.
+
+Many attributes may refer to the same attribute type.
+
+Each attribute type supports some general and special properties.
+
+**Akeneo PIM** provides built in set of attribute types:
+
+================================  ================  ==========================================================================================================
+Code                              Field             Description
+================================  ================  ==========================================================================================================
+pim_catalog_identifier            string            **(!!)** Only one attribute of this type is allowed. Product identification attribute: it may be **sku**, **erp product id** or any other, it may be unque or not unique.
```

Comment:
@jjanvier Hmmm... checked it! It is possible to set it 0 in attribute.csv and it will not throw any error but anyway after installation identifier is marked as unique. Thank you! I didn't mentioned it before at least in master version. ))